### ISSUE TITLE
-titleNs seems to be ignored

### ISSUE THREAD

**Issue by FabricioRocha** [gturri/nspages#139]
Hello, Guillaume!

I have just installed the current stable DokuWiki and nspages from its Extension Manager.

I am using nspages for a sidebar. I have tried a lot of things for getting rid of the "Pages in this namespace:" title or change it, but nothing worked.

Please, is there something wrong in this usage?

```<nspages -simpleList -h1 -r -exclude:[sidebar start] -textNs="Keywords">```

Also, but not so directly related, in "Configuration Settings" for the plugin I included _start_ and _sidebar_ in the list of "global exclude", but they kept being listed if I didn't use the _-exclude_ option as above.

Thank you very much!

Fabricio Rocha
Brasilia, Brasil

**Comment by gturri** [comment]
To get rid of `Pages in this namespace:` you should use the option `-textPages=""` (instead of `-textNs`).

For the "global exclude" configuration I'm not sure what is going wrong. It could be for instance because of whitespaces (ie: you should have `sidebar,start` in this conf, not `sidebar, start`).
Could you perhaps copy-paste here the content you put in this config?

**Comment by FabricioRocha** [comment]
Hello again, Guillaume!

Sorry for taking your time. I got confused about `-textNs` because of the tree example in the plugin page. To the point I did not notice the description of `textPages` just above...

You were straight to the point about the `globalexclude` spaces after commas. Just as I removed them, the pages I did not want listed went out of the sidebar.

Thank you very much for your prompt and precise response!

Fabricio Rocha
Brasilia, Brasil
